packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -383,10 +383,15 @@ export function updateAuthSignInSignOutUrl(cwd: string, settings: any): Promise<
     if (settings?.overrides?.category === 'auth') {
       chain.wait('A migration is needed to support latest updates on auth resources').sendConfirmYes();
     }
+    chain.wait('What do you want to do?');
+
+    if (settings?.skipThree) {
+      chain.sendKeyDown(3);
+    } else {
+      chain.sendKeyDown(2);
+    }
```

Comment:
what's the difference between 2 and 3 ?
Should this be `settings?.whatDoYouWantToDo = <some_enum>` ?

If you want to keep numbers maybe this should be `settings.whatDoYouWantToDoOption = 3` with default 2?